homeassistant/components/samsungtv/media_player.py

```diff
@@ -124,7 +125,16 @@ def get_remote(self):
         """Create or return a remote control instance."""
         if self._remote is None:
             # We need to create a new instance to reconnect.
-            self._remote = SamsungRemote(self._config.copy())
+            try:
+                self._remote = SamsungRemote(self._config.copy())
+            # This is only happening when the auth was switched to DENY
+            # A removed auth will lead to socket timeout because waiting for auth popup is just an open socket
+            except AccessDenied:
```

Comment:
Since we're catching the error here, there will not be a attribute error outside when trying to access the remote instance.

There are also error catching outside of this error that won't have an effect since we're catching it here.

Probably raise the error here again after starting the config flow.

Reply:
OK, the second is clear. But why the attribute error? Do I need to copy the config_entry?

Reply:
By default the remote will be `None`, right?

Reply:
Yes, `None` can always happen. But only in coincidence with an exception.

Reply:
Yes but we're catching that exception here, and the access is outside so `None` will become a problem.